sql/ddl_internal.sql

```diff
@@ -107,11 +101,35 @@ BEGIN
         END IF;
     END IF;
     IF actual_type != given_type THEN
-        RAISE EXCEPTION 'cannot call drop_chunks with a % on hypertables with a time type of: %', given_type, actual_type;
+        RAISE EXCEPTION 'cannot call % with a % on hypertables with a time type of: %', func_name, given_type, actual_type;
     END IF;
 END
 $BODY$;
 
+-- TODO rename function
```

Comment:
are both of the above functions needed?

Reply:
I can probably merge the two since the first implementation is no longer called directly from drop_chunks. will look into it